ci/travis/build-docker-images.py

```diff
@@ -142,24 +153,28 @@ def _build_cpu_gpu_images(image_name, no_cache=True) -> List[str]:
                     nocache=no_cache,
                     buildargs=build_args)
 
-                full_output = ""
+                full_output = []
                 try:
                     start = datetime.datetime.now()
                     current_iter = start
                     for line in output:
+                        full_output.append(line.decode("utf-8"))
                         if datetime.datetime.now(
                         ) - current_iter >= datetime.timedelta(minutes=5):
                             current_iter = datetime.datetime.now()
                             elapsed = datetime.datetime.now() - start
                             print(f"Still building {tagged_name} after "
                                   f"{elapsed.seconds} seconds")
-                        full_output += line.decode("utf-8")
+                            if elapsed >= datetime.timedelta(minutes=15):
+                                print("Build output:")
+                                print(*full_output, sep="\n")
+                                full_output = []
```

Comment:
Do we want to reset `start` here? Or are we okay with printing `Build output` every time?

Reply:
Added a comment and renamed full_output to cmd_output here. We clear the accumulated output (cmd_output) after printing, so no output line is printed twice.